CHANGELOG.md

```diff
@@ -1,3 +1,103 @@
+
+* Adds support for `EXPLAIN(ANALYZE, WAL)` for PG13
+
+* Adds support for `WITH TIES` option in SELECT and INSERT SELECT queries
+
+* Avoids taking multi-shard locks on workers
+
+* Enforces `citus.max_shared_pool_size` config in COPY queries
+
+* Enables custom aggregates with multiple parameters to be executed on workers
+
+* Enforces `citus.max_intermediate_result_size` in local execution
+
+* Improves cost estimation of INSERT SELECT plans
+
+* Introduces delegation of procedures that read from reference tables
+
+* Prevents pull-push execution for simple pushdownable subqueries
+
+* Sorts the output of `EXPLAIN ANALYZE` by execution duration.
+
+* Disallows `ON TRUE` outer joins with ref & dist tables when ref table is outer
+
+* Disallows field indirection in INSERT/UPDATE queries
+
+* Disallows volatile functions in UPDATE subqueries
+
+* Fixes ALTER TABLE IF EXISTS SET SCHEMA bug with non-existing tables
+
+* Fixes CREATE INDEX CONCURRENTLY crash with local execution
+
+* Fixes CREATE INDEX CONCURRENTLY with no index name crash with postgres tables
+
+* Fixes `citus_finish_pg_upgrade` to drop all backup tables
+
+* Fixes a bug that cause failures when `RECURSIVE VIEW` joined reference table
+
+* Fixes DROP SEQUENCE failures when metadata syncing is enabled
+
+* Fixes a bug that caused CREATE TABLE with CHECK constraint to fail
+
+* Fixes a bug that caused VACUUM to deadlock in PG13
+
+* Fixes master_update_node failure when no background worker slots are available
+
+* Fixes a bug that caused replica identity to not be propagated on shard repair
+
+* Fixes a bug that could cause crashes after connection timeouts
+
+* Fixes a bug that could cause crashes with certain compile flags
+
+* Fixes a bug that could cause deadlocks on CREATE INDEX
+
+* Fixes a bug with genetic query optimization in outer joins
+
+* Fixes a bug with lists of configuration values in ALTER ROLE SET statements
+
+* Fixes a crash because of overflow in partition id
+
+* Fixes a crash when aggregating empty tables
+
+* Fixes a crash with inserting domain constrained composite types
+
+* Fixes a crash with multi-row & router INSERT's in local execution
+
+* Fixes a memory leak in subtransaction memory handling
+
+* Fixes a possibility of doing cleanup more than once
+
+* Fixes incorrect setting of join related fields
+
+* Fixes memory issues around deparsing index commands
+
+* Fixes reference table access tracking for sequential execution
+
+* Fixes removal of a single node with only reference tables
+
+* Fixes sending commands to coordinator when it is added as a worker
+
+* Fixes write queries with const expressions and COLLATE in various places
+
+* Fixes wrong cancellation message about distributed deadlock
+
+* Forces aliases in deparsing queries with anonymous column references
+
+* Improves error message when creating a foreign key to a local table
+
+* Makes `citus_prepare_pg_upgrade` idempotent by dropping transition tables
```

Comment:
I would move this one up in the list, somewhere before all the various fixes.

Reply:
which one, Improves.. or Makes.. ?

Reply:
I meant "Makes ...", but the improves makes sense to move as well.

Reply:
I think both, done at a5f62e850cd228a3dbab8c8c6d32f38b611a58a7